lms/djangoapps/certificates/config.py

```diff
@@ -15,3 +15,18 @@
 # .. toggle_use_cases: open_edx
 # .. toggle_creation_date: 2017-09-14
 AUTO_CERTIFICATE_GENERATION = WaffleSwitch(f"{WAFFLE_NAMESPACE}.auto_certificate_generation", __name__)
+
+
+# .. toggle_name: SEND_CERTIFICATE_CREATED_SIGNAL
+# .. toggle_implementation: SettingToggle
+# .. toggle_default: False
+# .. toggle_description: When True, we will publish `CERTIFICATE_CREATED` signals to the event bus when a
+#   GeneratedCertificate instance with a "passing status" is created/updated. This signal will be consumed by the
+#   Credentials IDA in order to keep its records up to date. This is a temporary toggle while we are preparing the
+#   Credentials IDA to use the event bus. It should eventually be removed and this functionality should always be
+#   enabled.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2023-04-11
+# .. toggle_target_removal_date: 2023-07-31
```

Comment:
I think I may be a bit overzealous (or overly optimistic) with this timeline. Is it safe to assume that the event bus will be required for Open edX installations at some point in the future?

Reply:
We have been assuming the event bus will be required eventually, but eventually may be fairly far out. That said, publishing an event without an event bus configured is just a no-op right now so once we verify that it can work in prod it's not terribly dangerous to remove the toggle.